library/src/main/java/com/datatorrent/lib/stream/StreamMerger.java

```diff
@@ -41,35 +47,57 @@
 @Stateless
 public class StreamMerger<K> extends BaseOperator
 {
-  /**
-   * Data input port 1.
-   */
-  public final transient DefaultInputPort<K> data1 = new DefaultInputPort<K>()
+  private int portCount = 0;
+  private final transient ArrayList<DefaultInputPort<K>> ports = new ArrayList<>(portCount);
```

Comment:
No, there isn't. But it is possible to chain multiple mergers with 2 ports and accomplish the same.